assets/src/edit-story/components/library/panes/text/textSets/raw/editorial.json

```diff
@@ -0,0 +1,316 @@
+{
+  "current": "5f52f0ff-facb-448c-95f1-5153e3c20ad8",
```

Comment:
A direct copy and paste from the Editor's debug window so it'll be easy for a non-dev person to create the text sets and for them or any of us to copy it into the repo for use.